src/Usecases/UntestedParsers/CouncilTax/CTaxEnquiryForm.cs

```diff
@@ -13,6 +13,15 @@ class CTaxEnquiryForm : ILetterParser
     {
         public LetterTemplate Execute(string html)
         {
+            // Wrap up the council name into p HTML node so it could be targetted.
+            var cnCases = new Regex(@"(?<=>)\s*Hackney\s*(?=<i>Rev)|(?<=>)\s*Hackney\s*(?=</font>)", RegexOptions.Singleline);
+
+            html = cnCases.Replace(
+                    html,
+                    @"<b id=""council-name"">Hackney </b>",
```

Comment:
<strong> tags would be preferable for bold text, but this will work too

Reply:
Had no idea, will keep this in mind next time... unless a lot of such small things come out of this review to be worth another commit on this PR. :D